chatbot/model_service/builds/base/Containerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.access.redhat.com/ubi9/python-39:latest
+# MODEL_FILE should be set at runtime
+ENV MODEL_FILE=llama-2-7b-chat.Q5_K_S.gguf
+ENV MODEL_PATH=/locallm/models/${MODEL_FILE}
```

Comment:
Are these really needed here? We are mounting volume at runtime so we have to set them with the correct path/name 
Maybe we should add this kind of info somewhere in the ai-studio or agree that every model service from now on would need the env MODEL_PATH and the sample application MODEL_ENDPOINT. So we set them at runtime. If so, this has to be documented

Reply:
yes, I don't think we need these any longer as it will be required to define model_path at runtime instead.

Reply:
MODEL_PATH is used in the python code, yea - I figured I'd set them here as a default but it's probably better to remove them, so I will do that

Reply:
removed these